libs/intl/src/useLabel.tsx

```diff
@@ -22,13 +22,18 @@ import { CommonKeys } from "./IntlProvider";
 import { useIntlContext } from "./useIntlContext";
 
 export function useLabel(name: keyof CommonKeys): string {
-  const { dictionary, lang, commonKeys } = useIntlContext();
+  const intlContext = useIntlContext();
 
-  const key = commonKeys[name] || name;
+  if (intlContext) {
+    const { lang, dictionary, commonKeys } = intlContext;
+    const key = commonKeys[name] || name;
 
-  if (!dictionary?.translations?.[lang]?.[key] && !dictionary?.messages?.[key]) {
-    return defaultDictionary.translations?.[lang]?.[key] || defaultDictionary.messages[name];
+    if (!dictionary?.translations?.[lang]?.[key] && !dictionary?.messages?.[key]) {
+      return defaultDictionary.translations?.[lang]?.[key] || defaultDictionary.messages[name];
+    }
+
+    return (dictionary?.translations?.[lang]?.[key] || dictionary?.messages?.[key]) as string;
   }
 
-  return (dictionary?.translations?.[lang]?.[key] || dictionary?.messages?.[key]) as string;
+  return "Required field";
```

Comment:
Add a parameter to use Label of type string and return that value instead of hardcoded